src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java

```diff
@@ -133,6 +133,7 @@
      */
     @Advanced
     @Argument(fullName = MAX_MNP_DISTANCE_LONG_NAME, shortName = MAX_MNP_DISTANCE_SHORT_NAME,
-            doc = "Two or more phased substitutions separated by this distance or less are merged into MNPs.", optional = true)
+            doc = "Two or more phased substitutions separated by this distance or less are merged into MNPs. " +
+            "When used in conjunction GVCF mode, resulting GVCFs cannot be joint-genotyped.", optional = true)
```

Comment:
Maybe add a "WARNING:" prefix here to make this disclaimer stand out in the help output.